pyproject.toml

```diff
@@ -36,6 +36,13 @@ sphinxcontrib-napoleon = { optional = true, version = "*" }
 xlrd = { optional = true, version = "*" }
 xlsxwriter = { optional = true, version = "*" }
 
+sqlalchemy = { version = "^2.0.16", optional = true }
+sqlalchemy-access = { version = "^2.0.2", optional = true, platform = "win32" }
+
+matplotlib = { version = ">=3.7.1", optional = true }
+pyvis = { version = "^0.1.8", optional = true }
+plotly = { version = ">=5.15.0", optional = true }
+

```

Comment:
Look, I can leave comments on specific rows!

Reply:
Great!  That'll make it much easier to work out what you're actually talking about :p